addons/l10n_se_ocr/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name' : 'Sweden - Structured Communication OCR',
+    'version' : '1.0',
+    'author': 'XCLUDE',
+    'website': 'https://www.xclude.se',
+    'category': 'Localization',
+    'description': """
+Add Structured Communication to Customer Invoices and Vendor Bill.
+------------------------------------------------------------------
+
+Using OCR structured communication simplifies the reconciliation between invoices and payments.
+
+For Customer Invoicing support for OCR level 1 to 4 is added.
+
+For Vendor Bill support for Vendor Specific OCR is added and validation for OCR.
+
+    """,
+    'depends': ['account', 'l10n_se'],
```

Comment:
We can just make it depend on l10n_se alone.

Reply:
Fixed.